ui/src/components/Main.jsx

```diff
@@ -19,144 +16,70 @@ import WorkflowParametersDialog from '../containers/WorkflowParametersDialog';
 import GphlWorkflowParametersDialog from '../containers/GphlWorkflowParametersDialog';
 import SelectProposalContainer from '../containers/SelectProposalContainer';
 import diagonalNoise from '../img/diagonal-noise.png';
-import { resetChatMessageCount } from '../actions/remoteAccess';
-import { Widget, addResponseMessage, addUserMessage } from 'react-chat-widget';
 import { showDialog } from '../actions/general';
 import { LimsResultDialog } from './Lims/LimsResultDialog';
 import LoadingScreen from './LoadingScreen/LoadingScreen';
 import styles from './Main.module.css';
 
 import 'react-chat-widget/lib/styles.css';
 import './rachat.css';
-import { store } from '../store';
 import { getInitialState } from '../actions/login';
-import { fetchChatMessages, sendChatMessage } from '../api/remoteAccess';
 import MXNavbar from './MXNavbar/MXNavbar';
+import ChatWidget from './ChatWidget';
 
-class Main extends React.Component {
-  constructor(props) {
-    super(props);
-    this.handleClick = this.handleClick.bind(this);
-    this.handleNewUserMessage = this.handleNewUserMessage.bind(this);
-    this.onChatContainerClick = this.onChatContainerClick.bind(this);
-  }
-
-  componentDidMount() {
-    if (!this.props.applicationFetched) {
```

Comment:
I removed this condition, since when this component renders, `applicationFetched` is always `false`. If the user logs out, it gets unmounted before `applicationFetched` is set back to `false`.